File: fabric/src/main/java/io/github/techtastic/ccvs/fabric/PlatformUtilsImpl.java
package io.github.techtastic.ccvs.fabric;

import io.github.techtastic.ccvs.fabric.config.CCVSConfig;

public class PlatformUtilsImpl {
    public static boolean canTeleport() {
        return CCVSConfig.CAN_TELEPORT.get();
    }

    public static boolean isCommandOnly() {
        return CCVSConfig.COMMAND_ONLY.get();
    }

    public static boolean exposePhysTick() {
        return CCVSConfig.EXPOSE_PHYS_TICK.get();
    }
}

File: forge/src/main/kotlin/io/github/techtastic/cc_vs/forge/PlatformUtilsImpl.kt
package io.github.techtastic.cc_vs.forge

import io.github.techtastic.cc_vs.forge.config.CCVSConfig

object PlatformUtilsImpl {
    @JvmStatic
    fun canTeleport(): Boolean = CCVSConfig.CAN_TELEPORT.get()

    @JvmStatic
    fun isCommandOnly(): Boolean = CCVSConfig.COMMAND_ONLY.get()

    @JvmStatic
    fun exposePhysTick(): Boolean = CCVSConfig.EXPOSE_PHYS_TICK.get()
}